### ISSUE TITLE
checkout loader keeps loading not stop in magento 2.2.2

### ISSUE THREAD

**Issue by RahulBhut21** [magento/magento2#13398]
Hello, I am using **Magento 2.2.2** for learning Magento framework.
when I check out from purchase or cart with any product, checkout page start loading details
but checkout loader keeps loading.
![screenshot 5](https://user-images.githubusercontent.com/31387311/35468512-7cdba76e-0345-11e8-943d-9cd6803d5704.png)
Help Me, Thank You

**Comment by oliverklee** [comment]
@RahulBhut21 Do you get any JavaScript errors on the browser error console? Are there any network errors in the networking tab of the browser tools?

**Comment by RahulBhut21** [comment]
Yes @oliverklee  "304 Not Modified" error or warring come into networking tab for 30-35 js file.
And this was displayed in console of browser
(1)Loading failed for the <script> with source “http://localhost/Magento/pub/static/version1516704258/frontend/Magento/luma/en_US/Dotdigitalgroup_Email/js/trackingCode.js”.
checkout:1

(2)Error: Script error for tracking code
http://requirejs.org/docs/errors.html#scripterror

(3)JQMIGRATE: jQuery.browser is deprecated

**Comment by garoal123** [comment]
I have the same problem, and it shows me the following errors in the console:
![captura de pantalla 2018-01-30 a la s 12 47 47 p m](https://user-images.githubusercontent.com/35971956/35584880-f5e19722-05bb-11e8-9e2f-723bed80ecdc.png)

**Comment by garoal123** [comment]
I have the same problem, and it shows me the following errors in the console:
![captura de pantalla 2018-01-30 a la s 12 47 47 p m](https://user-images.githubusercontent.com/35971956/35585257-1783239a-05bd-11e8-9bb1-725642578220.png)


I am using version 2.1.5

**Comment by ananth-iyer** [comment]
@RahulBhut21 @garoal123 
Please check `pub/` and `pub/static/` directory for `.htaccess`

If you didn't find `.htaccess` then it's should be there because that file is responsible to load js, css and images from `pub/` directory

**Comment by teamloom** [comment]
@RahulBhut21 do you experience this issue in Safari and Chrome?

**Comment by RahulBhut21** [comment]
@teamloom haven't Safari and in Chrome product is not added to cart so check out process is not possible

**Comment by damac23** [comment]
The problem seems to be AdBlock-related.
uBlock Origin is blocking this URL /pub/static/version_____XY____/frontend/Magento/luma/de_DE/Dotdigitalgroup_Email/js/trackingCode.js based on a match for trackingCode.js

**Comment by RahulBhut21** [comment]
@damac23  you are right I stop uBlock Origin and it works now loader disables after page load.now I can do order using one page.
Thank you @damac23

**Comment by paginasweb** [comment]
i  just installed 2.2.3 and same issue, i dont have ad block, i dont have any network errors , its just a demo site, zero plugins or anything, weird


![capture](https://user-images.githubusercontent.com/2107317/36821181-a8ecdc4a-1cbf-11e8-9bb2-104e58b928c2.JPG)

**Comment by goncalotomas** [comment]
Same as @paginasweb, I have 2.2.3 and I have this issue, although in my case it goes away when I turn off uBlock Origin. This needs to be reviewed again.

**Comment by hostep** [comment]
Hi guys, just FYI:

This sounds like a duplicate of https://github.com/magento/magento2/issues/12828, which got fixed by https://github.com/magento/magento2/pull/13061 and the fix will most likely be included in Magento 2.2.4.

**Comment by php4umagento** [comment]
Same here with 2.2.6 and custom theme. Only happens in Firefox > 57 on Windows

**Comment by TomashKhamlai** [comment]
@magento-engcom-team give me 2.3.0 instance

**Comment by magento-engcom-team** [comment]
Hi @TomashKhamlai. Thank you for your request. I'm working on Magento 2.3.0 instance for you

**Comment by magento-engcom-team** [comment]
Hi @TomashKhamlai. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [ ] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [ ] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [ ] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by magento-engcom-team** [comment]
Hi @TomashKhamlai, here is your [Magento instance](https://i-13398-2-3-0.instances.magento-community.engineering).
Admin access: [https://i-13398-2-3-0.instances.magento-community.engineering/admin](https://i-13398-2-3-0.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by KilalaPhu** [comment]
> I have the same problem, and it shows me the following errors in the console:
> ![captura de pantalla 2018-01-30 a la s 12 47 47 p m](https://user-images.githubusercontent.com/35971956/35585257-1783239a-05bd-11e8-9bb1-725642578220.png)
> 
> I am using version 2.1.5

hi how fix it. hope you can help em, i error this bug.
@garoal123

**Comment by mhilko** [comment]
Checkout page seems to be one of the most painful things for Magento developers.
Fortunately, there is a great guide on this issue. It contains the most common problems developers usually have with it. Here is the link: [https://onilab.com/blog/magento-checkout-slow-fix-it-in-8-simple-steps/](https://onilab.com/blog/magento-checkout-slow-fix-it-in-8-simple-steps/) I hope you will find it valuable.

**Comment by OvalMedia** [comment]
Checkout is the most sensible part of a shop. And it has been effed up big time in M2. Totally over-engineered. I have set up 2 stores now and spent ~50% of my time hunting bugs and issues. Yet customers keep complaining that the checkout does not work for them and it is close to impossible to find out what the issues are. Well done magento, well done.

**Comment by aqdennis** [comment]
Issue still exists for me in version 2,3,2.  When I reached the payment page I just keep having the spinner going around. I can select a payment method and continue to the PSP though, but users get confused as they think they have to wait for the page finishing loading

![image](https://user-images.githubusercontent.com/48013601/80305064-bcd58c80-87ba-11ea-9ebe-ddd1fe74a87a.png)
